docs/tasks.rst

```diff
@@ -18,7 +18,7 @@ with.
 
  [General]
  name=/restraint/env/metadata
- owner=Bill Peck <bpeck@redhat.com>
+ owner=Happy Golucky <hgolucky@funland.com>
```

Comment:
This domain exists. Use example.com instead.

Reply:
I did make it up. I'll use example.come